appveyor.yml

```diff
@@ -1,16 +1,18 @@
 version: "{build}"
+image: Visual Studio 2017
```

Comment:
This image already contains `Go` (actually lots of versions :stuck_out_tongue:), see https://www.appveyor.com/docs/build-environment/#golang
We don't need to download/install it :wink: